c++/src/capnp/compat/websocket-rpc-test.c++

```diff
@@ -0,0 +1,73 @@
+
+#include "websocket-rpc.h"
+#include <kj/test.h>
+
+#include "json.capnp.h" // Arbitrary schema, just so we have something to play with
```

Comment:
Usually we'd use `test.capnp`, especially `TestAllTypes`. See also `test-util.h` which has a bunch of methods for filling in the content and checking it later.